lib/strategy.js

```diff
@@ -44,14 +44,14 @@ Examples:
 **/
 function Strategy(options, verify) {
   options = options || {};
-  options.authorizationURL = options.authorizationURL || 'http://connect.sheknows.com/authorize';
-  options.tokenURL = options.tokenURL || 'http://connect.sheknows.com/token';
+  options.authorizationURL = options.authorizationURL || 'http://atlas-sso.revtech.internal/oauth/authorize';
+  options.tokenURL = options.tokenURL || 'http://atlas-id-manager-sso/oauth/token';
```

Comment:
Let's default all of these to the public prod URLs: https://atlas-sso.pmc.com/...

Reply:
Done